packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/appSync-walkthrough.ts

```diff
@@ -393,8 +394,7 @@ const updateApiInputWalkthrough = async (context: $TSContext, project: $TSObject
 
 export const serviceWalkthrough = async (context: $TSContext, serviceMetadata: $TSObject) => {
   const resourceName = resourceAlreadyExists();
-  const providerPlugin = await import(context.amplify.getProviderPlugins(context)[providerName]);
-  const transformerVersion = await providerPlugin.getTransformerVersion(context);
+  const transformerVersion = await ApiCategoryFacade.getTransformerVersion(context);
```

Comment:
Don't we need `getProviderPlugins` call anymore?

Reply:
We do, we still leverage this functionality, but it's wrapped in the Facade layer instead of living here.